data.sql

```diff
@@ -1,5 +1,4 @@
-/* Populate database with sample data. */
-
-INSERT INTO animals (name) VALUES ('Luna');
-INSERT INTO animals (name) VALUES ('Daisy');
-INSERT INTO animals (name) VALUES ('Charlie');
+INSERT INTO animals VALUES(1,'Agumon','2020-02-03',0,'TRUE',10.23);
+INSERT INTO animals VALUES(2,'Gabumon','2018-11-15',2,'TRUE',8);
+INSERT INTO animals VALUES(3,'Pikachu','2021-01-07',1,'FALSE',15.04);
+INSERT INTO animals VALUES(4,'Devimon','2017-05-12',5,'TRUE',11);
```

Comment:
- Optional
I would recommend removing the id field from the inserted data. The `Id` field is generated automatically by the database if you mark it as `Primary key`. Optionally you can also use `NOT NULL IDENTITY(1,1) which will automatically increment the id's as you insert more data.